MastodonSDK/Sources/MastodonUI/View/Content/MediaView+Configuration.swift

```diff
@@ -243,3 +244,82 @@ extension MediaView {
         return configurations
     }
 }
+
+extension MediaView {
+    public static func configuration(status: Mastodon.Entity.StatusEdit) -> [MediaView.Configuration] {
+        func aspectRatio(from attachment: Mastodon.Entity.Attachment) -> CGSize {
+            guard let width = attachment.meta?.width, let height = attachment.meta?.height else {
+                return .zero
+            }
+            return CGSize(width: width, height: height)
+        }
```

Comment:
This is new, right? Good idea!

Reply:
Yes but it was flawed and actually led to the crash as `CGSize.zero` is illegal in this case. I've fixed this.